R/proportional_hazards_data.R

```diff
@@ -282,11 +282,23 @@ coxnet_train <- function(formula,
   if (has_strata(formula, data)) {
     check_strata_nterms(formula, data)
     strata <- get_strata_glmnet(formula, data)
-    data_obj$y <- glmnet::stratifySurv(data_obj$y, strata = strata)
+
+    stratify_surv_call <- rlang::call2("stratifySurv", .ns  = "glmnet")
```

Comment:
If a package is in Suggests, like glmnet, you shouldn't have to do this.

You only have to do this if the package isn't listed as a dependency of your package at all

Reply:
Like in parsnip we have a `rand_forest()` engine for the `"randomForest"` package. That isn't listed as a dependency of parsnip at all though, so we have to use call2 there